config.yml

```diff
@@ -4,11 +4,15 @@
 # Current supported terminals: alacritty, urxvt, kitty, wezterm.
 #backend: urxvt
 
-# path to backend executable file
+# Path to backend executable file
 # NOTE: requires a backend key
 # NOTE: old key exe_path is now deprecated but can still be used
 #term_exe_path: /path/to/urxvt
 
+# Configuration file for the selected terminal.
+# This option is not supported by urxvt.
```

Comment:
Could you please add a comment to mention when this is set, the `load_term_conf` won't take effect anymore?